### PR TITLE
feat: format prisma pre-commit

### LINKED ISSUE #338: format prisma in pre-commit hook
**Is your feature request related to a problem? Please describe.**
It would be nice if the prisma files were formatted on commit

**Describe the solution you'd like**
Run `nx run db:prisma format` in the `pre-commit` file. 

**Describe alternatives you've considered**
We could just run `prisma format` manually or add a VSCode extension. 

**Additional context**

### PR DESCRIPTION
**Pull request by tycebrown**
## What has changed
- `nx run db:prisma format` is run before each commit. The formatting changes applied by this command are then added to the commit. 

## Connected Issues

closes #338

## QA Steps

1. Switch your local branch to this branch (`tyce-338-format-prisma`).
2. Locally, add spacing (tabs, spaces, etc.) to `schema.prisma` so that the file is not properly formatted.
   - If you have a VSCode extension which autoformats prisma files on save, you may need to disable that extension or temporarily change the "Language Mode" of `schema.prisma` from "Prisma" to "Plain Text" to test this PR.
3. Commit your changes.
4. Observe the diff of the commit. Confirm that there are no changes observed (since the formatting has been corrected).

## Post-Deployment

- [ ] Nothing required

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,3 +2,7 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npx lint-staged --relative
+
+nx prisma db format
+# Add the changes made by prisma format to the commit
+git add ./packages/db/src/schema.prisma
```
Comment (arrocke) [inline .husky/pre-commit]: We should use lint staged for this instead. That will automatically stage the changes and only run if the prisma file actually changed in the commit

File: .gitignore [hunk 2]
```diff
@@ -41,6 +41,9 @@ Thumbs.db
 # Next.js
 .next
 
+# NX Cache
+/.nx/cache
```
Comment (arrocke) [inline .gitignore]: This should already be covered by `.nx/` below

**Review by arrocke — CHANGES_REQUESTED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (tycebrown) [reply .husky/pre-commit]: Done.

File: .gitignore (hunk 2 above)
Comment (tycebrown) [reply .gitignore]: Done.

**Review by arrocke — APPROVED** [review]
Thanks for taking care of this

**Pull request merged**